pyrato/parametric.py

```diff
@@ -151,3 +151,36 @@ def air_attenuation_coefficient(
             ) * 20.0 / np.log(10.0) / (np.log10(np.exp(1.0)) * 10.0))
 
     return air_abs_coeff
+
+def calculate_sabine_reverberation_time(surfaces, alphas, volume):
+
+    """Calculate the reverberation time using Sabine's equation.
+
+    Calculation according to [#]_.
+
+    Parameters
+    ----------
+    surfaces : ndarray, double
+        Surface areas of all surfaces in the room in square meters
+    alphas : ndarray, double 
+        Absorption coefficients corresponding to each surface
+    volume : double
+        Room volume in cubic meters
+    
+    Returns
+    -------
+    reverberation_time_sabine :  double
+        The value of calculated reverberation time in seconds
+
+    References
+    ----------
+    .. [#] H. Kuttruff, Room acoustics, 4th Ed. Taylor & Francis, 2009.
+
+    """
+    alphas = np.asarray(alphas)
+    surfaces = np.asarray(surfaces)
```

Comment:
Here it may be beneficial to check if the size of both arrays are equal to catch potential user input errors.
```suggestion
    alphas = np.asarray(alphas)
    surfaces = np.asarray(surfaces)
    if surfaces.shape != alphas.shape:
        raise ValueError("The shapes of alphas and surfaces do not match.")
```
This information also would be good to have in the docstring..